test/e2e/kubectl/kubectl.go

```diff
@@ -514,8 +514,28 @@ var _ = SIGDescribe("Kubectl client", func() {
 			runOutput := framework.NewKubectlCommand(nsFlag, "run", "run-test", "--image="+busyboxImage, "--restart=OnFailure", "--attach=true", "--stdin", "--", "sh", "-c", "cat && echo 'stdin closed'").
 				WithStdinData("abcd1234").
 				ExecOrDie()
-			Expect(runOutput).To(ContainSubstring("abcd1234"))
-			Expect(runOutput).To(ContainSubstring("stdin closed"))
+
+			g := func(pods []*v1.Pod) sort.Interface { return sort.Reverse(controller.ActivePods(pods)) }
+			runTestPod, _, err := polymorphichelpers.GetFirstPod(f.ClientSet.CoreV1(), ns, "run=run-test", 1*time.Minute, g)
+			if err != nil {
+				os.Exit(1)
+			}
```

Comment:
Why don't we use
`Expect(err).To(BeNil())`
to know the actual err message?

Reply:
Fixed